include/z5/types/types.hxx

```diff
@@ -144,6 +144,7 @@ namespace types {
 
         static InverseCompressorMap & compressorToZarr() {
             static InverseCompressorMap cMap({{
+                {raw, "raw"},
```

Comment:
Good catch. I think I forgot to put this here because zarr encodes no compression as `null` instead of `raw`.
In any case, right now [this line](https://github.com/constantinpape/z5/blob/addb858ac143cfc82f823618d759abd8ede151cc/include/z5/filesystem/dataset.hxx#L133) will throw for zarr and no compression, so it's good to fix this!